blaze/compute/sql.py

```diff
@@ -1022,10 +1022,10 @@ def compute_up(expr, data, **kwargs):
     names = data.c.keys()
     assert names == expr._child.fields
     d = dict(zip(names, getattr(data, 'inner_columns', data.c)))
-    return sa.select(
-        d[col].label(new_col) if col != new_col else d[col]
-        for col, new_col in zip(expr._child.fields, expr.fields)
-    )
+    return reconstruct_select([d[col].label(new_col) if col != new_col
```

Comment:
I find ternary expressions harder to read when they are split like this, could we just break earlier like
```python
return reconstruct_select(
    (d[col].label(new_col) if col != new_col else d[col]
     for col, new_col in zipl(...),
    data,
)
```